src/track-versions.js

```diff
@@ -46,15 +46,15 @@ const updateReleaseCards = (logger, context, masterRepo, versionKey, version) =>
     .then(pagePromises => Promise.all(pagePromises))
 }
 
-module.exports = (robot) => {
+export default (robot) => {
   const logger = robot.log.child({name: 'track-versions'})
 
   robot.on(['push'], (context) => {
     const {payload} = context
     const branch = payload.ref.replace(/^refs\/heads\//, '')
     const repo = payload.repository.full_name
     const versionKey = repo
-    const version = payload.after.substring(0, 7)
+    const version = payload.after
```

Comment:
:shipit:

Reply:
yea in other news i learned that you can't use short refs when interacting with remotes so for my downstream automation i had to change this to include the whole sha